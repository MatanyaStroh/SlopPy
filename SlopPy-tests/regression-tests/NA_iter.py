# this should NOT result in an infinite loop, because iterating over an
# NA object should terminate instantly

for line in open(NON_EXISTENT_FILENAME):
  print line

