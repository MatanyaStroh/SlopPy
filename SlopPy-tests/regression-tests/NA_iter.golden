{NA <type 'exceptions.NameError'> "name 'NON_EXISTENT_FILENAME' is not defined"
 Traceback (most recent call last):
  File "regression-tests/NA_iter.py", line 4, in <module>
    for line in open(NON_EXISTENT_FILENAME):

Locals: {'__builtins__': <module '__builtin__' (built-in)>, '__name__': '__main__', '__file__': 'regression-tests/NA_iter.py', '__doc__': None, '__package__': None}
}
{NA <type 'exceptions.TypeError'> 'coercing to Unicode: need string or buffer, NA found'
 Traceback (most recent call last):
  File "regression-tests/NA_iter.py", line 4, in <module>
    for line in open(NON_EXISTENT_FILENAME):

Locals: {'__builtins__': <module '__builtin__' (built-in)>, '__name__': '__main__', '__file__': 'regression-tests/NA_iter.py', '__doc__': None, '__package__': None}
}
{NA_iternext | regression-tests/NA_iter.py <module>:4}
